stl/inc/format

```diff
@@ -2859,122 +2862,142 @@ _NODISCARD auto make_wformat_args(const _Args&... _Vals) {
     return _Format_arg_store<wformat_context, _Args...>{_Vals...};
 }
 
-template <class _Out, class _CharT>
-using format_args_t = basic_format_args<basic_format_context<_Out, _CharT>>;
+template <class _CharT, class... _Args>
+struct _Fmt_basic_string {

```

Comment:
P2216R3 uses name basic-format-string. Renaming this to `_Basic_format_string` would be more conventional.

Reply:
Which is a bit curious because it is a string_view

Reply:
It's not a `string_view` but a format string. Whether it's a view or not is irrelevant because the string is static.

Reply:
Changed to `_Basic_format_string`.